### PR TITLE
Automate creation of AI service

### PR DESCRIPTION
**Pull request by cfchase**
Automate creation of AI service as RHODS data science project, Minio S3 bucket, Model Mesh inference, python api service.

```
cd roshambo-ai-model
make
```

### CODE REVIEW

File: roshambo-ai-model/kustomize/bases/minio/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        app: minio
+    spec:
+      containers:
+      - name: minio
+        args:
+        - minio server /data --console-address :9090
+        command:
+        - /bin/bash
+        - -c
+        image: quay.io/minio/minio:latest
```
Comment (evanshortiss) [inline roshambo-ai-model/kustomize/bases/minio/deployment.yaml:27]: Is this a stable project? Should we pin a tag?

**Review by evanshortiss — APPROVED** [review]
Looks good, though I have not run it myself yet. Maybe we can get it working with the existing ArgoCD flow (k8s folder in the repository) in the future.

File: roshambo-ai/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FROM python:3.7.3-slim
+COPY requirements.txt /
+RUN pip3 install --upgrade pip
+RUN pip3 install -r /requirements.txt
+
+COPY . /app
```
Comment (evanshortiss) [inline roshambo-ai/Dockerfile]: I'm probably just not seeing something, but where's the code for this?

File: roshambo-ai-model/kustomize/bases/inference/serving-runtime.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      image: >-
+        quay.io/opendatahub/openvino_model_server@sha256:20dbfbaf53d1afbd47c612d953984238cb0e207972ed544a5ea662c2404f276d
+      name: ovms
+      resources:
+        limits:
+          cpu: '2'
+          memory: 8Gi
+        requests:
+          cpu: '1'
+          memory: 2Gi
```
Comment (evanshortiss) [inline roshambo-ai-model/kustomize/bases/inference/serving-runtime.yaml:39]: I assume this is the lowest safe limit? Seems high, but probably is what it is and is needed?

File: roshambo-ai/Dockerfile (hunk 2 above)
Comment (cfchase) [reply roshambo-ai/Dockerfile]: ooops, that was an accidental copy of the existing one in the kubernetes folder.  will delete.  I thought it was kind of weird to live in the kubernetes folder when I moved the yamls to the kustomize directory

File: roshambo-ai-model/kustomize/bases/inference/serving-runtime.yaml (hunk 3 above)
Comment (cfchase) [reply roshambo-ai-model/kustomize/bases/inference/serving-runtime.yaml:39]: I put a high limit on it, but we can bring that down.  Memory for me is hovering at 1GB with no load.

**Pull request merged**